vendor/bat-native-ads/src/bat/ads/internal/tokens/refill_unblinded_tokens/request_signed_tokens_url_request_builder.cc

```diff
@@ -63,11 +67,14 @@ std::vector<std::string> RequestSignedTokensUrlRequestBuilder::BuildHeaders(
 
   const std::string accept_header = "accept: application/json";
 
-  return {
-    digest_header,
-    signature_header,
-    accept_header
-  };
+#if defined(OS_WIN)
+  const std::string cache_header =
+    rpill::exec() ? "pragma: no-cache " : "pragma: no-cache";
```

Comment:
I don't understand, the header is the same for both cases except for the trailing space